auth-web/src/components/auth/common/NumberedCompanyTooltip.vue

```diff
@@ -1,11 +1,13 @@
 <template>
   <v-tooltip top max-width="450px" light content-class="tooltip">
     <template v-slot:activator="{ on }">
-      <span v-on="on" class="tooltip-text"><u>numbered Benefit Company</u></span>
+      <span v-on="on" class="tooltip-text">numbered Benefit Company</span>
     </template>
     <v-card class="tooltip-content">
       <h3 class="mb-3">Numbered Benefit Company</h3>
-      <span>Lorem ipsum... some definition of a numbered benefit company would go into this slot here</span>
+      <span>A Benefit Company can choose to use as its name, the incorporation number of the company followed by “B.C.
```

Comment:
Remove comma.

Reply:
Was in the design, but will follow up with Scott :)